
Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-ttbh2[0m
{"level":"warn","ts":1581731826.977368,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581731826.9776685,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-62j2g[0m
{"level":"warn","ts":1581731827.5061762,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581731827.507147,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-trietlu-flask-jx-sample-pr-10-s-vcnt6-qvxh6[0m
{"level":"warn","ts":1581731832.563565,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581731832.8613648,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*******/flask-**-sample.git @ master in path /workspace/source"}

Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:93da7d63322cafdc0341e9ce88fa7ece7207667c,10:578fdddbbd6c0ae347d6a47d1154985bc1473464
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git fetch origin --unshallow 578fdddbbd6c0ae347d6a47d1154985bc1473464: 93da7d63322cafdc0341e9ce88fa7ece7207667c:
DEBUG: ran git fetch --unshallow origin 578fdddbbd6c0ae347d6a47d1154985bc1473464: 93da7d63322cafdc0341e9ce88fa7ece7207667c: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 93da7d63322cafdc0341e9ce88fa7ece7207667c
DEBUG: ran git reset --hard 93da7d63322cafdc0341e9ce88fa7ece7207667c in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git merge 578fdddbbd6c0ae347d6a47d1154985bc1473464
DEBUG: ran git merge 578fdddbbd6c0ae347d6a47d1154985bc1473464 in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git log --format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e 93da7d63322cafdc0341e9ce88fa7ece7207667c..HEAD
Merged SHA 9629fa1f1b44fd17147d89ef2041b8255f06c3cc with commit message 'Merge commit '578fdddbbd6c0ae347d6a47d1154985bc1473464'' into base branch master
Merged SHA 578fdddbbd6c0ae347d6a47d1154985bc1473464 with commit message 'Update flask_app.py' into base branch master
Merged SHA 0a9dc587148a0a3715e875a37797db5df0025e75 with commit message 'Update flask_app.py' into base branch master

Showing logs for build [32mtrietlu-flask-jx-sample-pr-10-s-vcnt6-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-code-scan[0m
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting python-taint
  Downloading https://files.pythonhosted.org/packages/5c/79/5b70d3299cb7dea63e2b015651c5c7507d723911ff2837e104ebd7079491/python_taint-0.42-py3-none-any.whl (52kB)
Installing collected packages: python-taint
Successfully installed python-taint-0.42
1 vulnerability found:
Vulnerability 1:
File: flask_app.py
 > User input at line 17, source "request.args.get(":
	 ~call_1 = ret_request.args.get('param', 'not set')
Reassigned in:
	File: flask_app.py
	 > Line 17: param = ~call_1
File: flask_app.py
 > reaches line 22, sink "replace(":
	~call_5 = ret_html.replace('{{ param }}', param)

[31m
Pipeline failed on stage 'from-build-pack' : container 'step-build-code-scan'. The execution of the pipeline has stopped.[0m
